be/src/olap/schema_change.cpp

```diff
@@ -1700,10 +1582,10 @@ OLAPStatus SchemaChangeHandler::_create_new_tablet(
 OLAPStatus SchemaChangeHandler::schema_version_convert(
```

Comment:
If you have duplicated rowset to handle ingestion when schema change, this function is redundant.